.github/CONTRIBUTING.md

```diff
@@ -80,33 +80,76 @@ Each contributor is required to [sign a CLA](https://cla.shopify.com/). This pro
 
 ### Contribution prerequisites
 
-- You have Node installed at v10.13.0+ and Yarn at v1.10.1+
+- You have Node installed at v16.13.0+ and Yarn at v1.22.18+
 
 ### Best practices
 
-Use the [documentation directory](https://github.com/Shopify/polaris-react/tree/main/documentation) for guidance on: [accessibility testing](https://github.com/Shopify/polaris-react/blob/main/documentation/Accessibility%20testing.md) and [adding to the change log](https://github.com/Shopify/polaris-react/blob/main/documentation/Versioning%20and%20changelog.md). For code style best practices, take a look at existing components.
+Use the [documentation directory](https://github.com/Shopify/polaris/tree/main/documentation) for guidance on: [accessibility testing](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md) and [adding to the changelog](https://github.com/Shopify/polaris/blob/main/documentation/Versioning%20and%20changelog.md). For code style best practices, take a look at existing components.
 
 Making sure the voice and tone is consistent across Polaris is important to us, for more information visit the [voice and tone guidelines](https://polaris.shopify.com/content/voice-and-tone).
 
 ### Development workflow
 
-After cloning Polaris React, run `yarn` to fetch its dependencies. Then you can run several commands:
+After cloning Polaris, run `yarn` to fetch its dependencies. Then you can run several commands:
```

Comment:
@aaronccasanova, @kyledurand  I plan to continue to iterate on these docs after addressing #5433 ✍️ 

Anything blocking this merge?

Reply:
Yalc docs looks great 🙌